blog/Release/2017-04-25_FreshRSS_1.6.3_is_available.md

```diff
@@ -0,0 +1,34 @@
+Title: FreshRSS 1.6.3 is available
+Date: 2017-04-25
+Comments_at: https://github.com/FreshRSS/freshrss.org/issues/33
+
+Hey community!
+
+FreshRSS has recently released it's 1.6.3 update !
+
+What happend since our last blog post about 1.2.0 ?
+A lot, FreshRSS can now:
+
+* Support experimentally PostgreSQL
+* FreshRSS supports now PHP 7 and requires at least PHP 5.3.3+
```

Comment:
For the record: Full (non-experimental) support of PostgreSQL from FreshRSS version 1.7